modules/apps/commerce/commerce-product-type-virtual-web/src/main/resources/META-INF/resources/js/TermsOfUseInputUtil.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+export default function ({
+	noneText,
+	portletNamespace,
+	selectWebContent,
+    termsOfUseJournalArticleBrowserURL,
```

Comment:
for nonText and selectWebContent, I don't think we need to pass them to the module. We can probably use `Liferay.Language.get('select-web-content')` and `Liferay.Language.get('none')`